### ISSUE TITLE
Rename all *ubuntu* packages

### ISSUE THREAD

**Issue by mariogrip** [ubports/ubuntu-touch#793]
Since we want to get these packages into different distros, we need to remove any ubuntu name from all our packages.

- [ ] ubuntu-ui-toolkit -> qml-module-suru-ui-toolkit       # qml-module-* is a convention used in Ubuntu archives (ref. https://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=qml-module&searchon=names )
- [ ] ubuntu-ui-extras -> qml-module-suru-ui-extras
- [ ] ubuntu-keyboard -> qtunity8-keyboard
- [ ] ubuntu-push -> ubports-push        # because ubports provides the service
- [ ] ubuntu-app-launch -> unity8-app-launch
- [ ] ubuntu-download-manager -> unity8-download-manager
- [ ] ubuntu-printing-app -> unity8-printing-app
- [ ] apparmor-easyprof-ubuntu -> apparmor-easyprof-click
- [ ] ubuntu-location(-provider...) -> unity8-location(-provider...)
- [ ] qtubuntu -> qtmirclient
- [ ] ubuntu-system-settings -> unity8-system-settings
- [ ] ubuntu-[coreappname]-app -> unity8-[coreappname]-app        # including e.g. `music-app` pkg, which becomes `unity8-music-app`

**Comment by dobey** [comment]
I don't think this makes a lot of sense currently. Ubuntu is the origin of the code, and it makes sense. Down the path of changing this lies madness, as if you're going to change the code to match the new naming scheme as well, you're going to run into a lot of things breaking.

**Comment by tsimonq2** [comment]
I respectfully disagree with you, @dobey. I think we should rename it so when other upstreams want to use it, they don't have to keep an Ubuntu name.

The long term goal is to just maintain packaging in salsa.debian.org anyway, so it makes sense.

**Comment by tsimonq2** [comment]
Wrong button. :wink:

**Comment by dobey** [comment]
Except unless you're also changing all the code throughout the whole stack, it's still there. The dbus services are still com.ubuntu and com.canonical, among other things. Quibbling over the package name seems a bit trite.

**Comment by tsimonq2** [comment]
I don't think it needs to be a blocker, but it certainly needs to be done eventually.

**Comment by UniversalSuperBox** [comment]
If we want the packages in Debian, they need to be renamed. I don't think we need to clear the dbus namespace of Ubuntu, but we will eventually. This ticket is only about the package namespace, please stay on topic.

**Comment by PhoenixLandPirate** [comment]
Could having "unity8" in the user space bring issues with other distros and use cases?

I dont think it necessarily does/will, but if youre using the unity8 keyboard on plasma mobile, the package name tells you that its meant for a different de.

That could be a positive as well, and I dont think its a big issue, just something to consider.

**Comment by rafael2k** [comment]
My list of missing packages in Debian 10 (after installing mir): is this one:
settings-ubuntu-schemas
libandroid-properties-dev
libconnectivity-qt1-dev
libgeonames-dev
libhardware-dev
libqmenumodel-dev
libsystemsettings-dev
libubuntu-app-launch3-dev
libubuntu-download-manager-common-dev libubuntugestures5-dev
libubuntugestures5-private-dev
libudm-common-dev
libunity-api-dev
libusermetricsoutput1-dev
qml-module-qtsysteminfo
qml-module-ubuntu-components | qml-module-ubuntu-components-gles
qtdeclarative5-ubuntu-content1
qtdeclarative5-ubuntu-settings-components

My mir packages are here:
deb http://www.telemidia.puc-rio.br/~rafaeldiniz/public_files/unity8 buster main
key here: http://www.telemidia.puc-rio.br/~rafaeldiniz/public_files/unity8/rafaeldiniz.gpg.key

**Comment by sunweaver** [comment]
@rafael2k: the URL is unreachable from here (Germany). I tried accessing it some minutes ago. (4pm CET, 2019-12-17).

**Comment by rafael2k** [comment]
> @rafael2k: the URL is unreachable from here (Germany). I tried accessing it some minutes ago. (4pm CET, 2019-12-17).

It must be a power shortage, but should be back soon.
